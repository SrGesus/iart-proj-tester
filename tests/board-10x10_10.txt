VC	VD	FC	VE	LV	VD	VC	LH	FB	FB
LV	FC	LH	FD	FD	BD	BD	LH	LH	BD
LH	FB	VB	BC	BC	BC	LV	LH	FD	FC
BC	BE	LH	BB	BB	BB	LH	BD	BE	FE
FE	FC	VD	BD	VE	LH	VB	LH	VE	FC
FC	VB	BE	VC	BE	VD	FE	VC	BD	VB
LH	VB	BD	FE	FD	BB	FB	FC	VD	LV
LV	VE	VC	FE	LV	BE	LH	FE	VE	BB
LV	VE	BB	BC	VD	FE	VB	VB	VE	LV
VC	VE	FE	VE	VD	FE	VC	BD	FC	FB
