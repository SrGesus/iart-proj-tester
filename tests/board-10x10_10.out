VB	VE	FB	VB	LH	VE	VB	LH	FE	FB
LV	FC	LV	FC	FB	BD	BC	LH	LH	BE
LV	FB	VD	BB	BE	BD	LH	LH	FE	FC
BD	BC	LH	BE	BD	BC	LH	BB	BB	FE
FC	FB	VB	BE	VD	LH	VE	LV	VD	FE
FB	VD	BE	VD	BB	VE	FC	VD	BB	VE
LV	VB	BC	FE	FC	BD	FE	FD	VC	LV
LV	VD	VE	FD	LH	BC	LH	FE	VB	BE
LV	VB	BC	BB	VE	FD	VE	VB	VC	LV
VD	VC	FD	VC	VD	FE	VD	BC	FE	FC
